julia.html.markdown

```diff
@@ -798,12 +798,12 @@ code_native(square_area, (Float64,), syntax = :intel)
 # Note that julia will use floating point instructions if any of the
 # arguments are floats.
 # Let's calculate the area of a circle
-circle_area(r) = pi * r * r     # circle_area (generic function with 1 method)
-circle_area(5)  # 78.53981633974483
+circleArea(r) = pi * r * r     # circleArea (generic function with 1 method)
+circleArea(5)  # 78.53981633974483
 
-code_native(circle_area, (Int32,), syntax = :intel)
+codeNative(circleArea, (Int32,), syntax = :intel)
     #         .text
-    # ; Function circle_area {
+    # ; Function circleArea {
```

Comment:
Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194